src/sessions/Coordinator.js

```diff
@@ -56,6 +73,7 @@ class Coordinator {
 
   @observable patients = patients
 
+  // here is where we would connect to our DB and get REAL patient information
```

Comment:
Looking forward to it!

I should be able to hook up the rest of the data after this is merged in.